style.css

```diff
@@ -385,6 +388,57 @@ textarea:-ms-input-placeholder {
   margin-bottom: 8px;
 }
 
+/* MOBILE MENU */
+
+.mobile-menu-style {
```

Comment:
- [ ] Kindly make sure you stick to the original template as required in the requirement, the spacing, colors, typo are supposed to be similar to the original design in Figma. As you may notice how your color is a little bit not visible because of the `blur` color of your text in the background. Please make sure you adjust that so that the user would still see the link clearly for a better UX.

### Your design :point_down: 
![image](https://github.com/HFG43/Portfolio_Mobile_First/assets/97100091/751e7efb-7b70-441f-af8d-c3f74e4dfab1)
![image](https://github.com/HFG43/Portfolio_Mobile_First/assets/97100091/d3599ae3-cbb0-4ccd-be86-f681102e303b)

### Figma template :point_down: 
![image](https://github.com/HFG43/Portfolio_Mobile_First/assets/97100091/c511ccbc-ba3c-4415-a1c2-783ad011038b)